### PR TITLE
Update base container for website

### PR DESCRIPTION
**Pull request by gagb**
## Related issue number
#1502 

## Checks

- [ ] I've included any doc changes needed for https://microsoft.github.io/autogen/. See https://microsoft.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ ] I've made sure all auto checks have passed.

### CODE REVIEW

**Comment by gagb** [comment]
@jackgerrits @victordibia -- I have made the fixes but the documentation issue is still there. But its not blocking me so I haven't addressed that here.

File: website/build_website.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+pydoc-markdown
+
+# Render the website using Quarto
+quarto render ./docs
+
+# Process notebooks using a Python script
+python ./process_notebooks.py
+
+# Start the website using yarn
+yarn start
```
Comment (jackgerrits) [inline website/build_website.sh:17]: While this is definitely helpful, it adds another place to keep in sync

**Review by jackgerrits — APPROVED** [review]

**Review by victordibia — APPROVED** [review]

File: website/build_website.sh (hunk 1 above)
Comment (gagb) [reply website/build_website.sh:17]: It does. How about we keep this up to date and speed us up. And fix the redundant text in website/readme and contribute.md. That way there are only two things:
- script 
- documentation of the script

Thoughts?

Comment (jackgerrits) [reply website/build_website.sh:17]: Yeah that sounds good!

**Pull request merged**